webpack.config.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
   },
   plugins: [
     new HtmlWebpackPlugin({
-     template: './src/index.html'
+      template: './src/index.html',
     }),
   ],
```

Comment:
- [ ] In this webpack configuration file there is a file called `print.js` am not sure why you are using it here and when I run `npm run build` it is showing an error. I was able to run the server when I removed it

Reply:
- [ ] Please write your README, add some information like a short description etc, and don't forget to add how to get the app locally up and running